src/ProductTour/README.md

```diff
@@ -52,6 +52,10 @@ The checkpoint objects themselves have additional props that can override the pr
           placement: 'bottom',
           target: '#checkpoint-3',
           title: 'Third checkpoint',
+          onEnd: () => {
+            console.log('Ended the third checkpoint');
+            setIsTourEnabled(false);
+          } // Override the default onEnd above
```

Comment:
[curious] Is it really overriding the default `onEnd` above or is it in addition to the default `onEnd` above as is?